tests/scripts/start-storybook.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+npx kill-port 6006
```

Comment:
was the issue that both visual-coverage.sh and start-storybook.sh both started up storybook so they would port clobber each other?

Reply:
Since in visual-coverage.sh it is running the script ./start-storybook.sh, I don't think storybook is started two times.

Reply:
Oh gotcha, how did this issue come up? Was it during local dev?

Reply:
yeah this is something that happened when I had storybook running locally and then also ran the test command locally

Reply:
I feel like in that case the right behavior would be to allow the second storybook to start in a different port, and use that for the test command? Alternatively we could say that if something is already running on port 6006, assume that it's a preeixsting storybook server and use that instead of starting a new one?